packages/ketcher-react/src/script/ui/views/modal/components/document/Save/Save.jsx

```diff
@@ -274,29 +280,47 @@ class SaveDialog extends Component {
     const { filename, format } = formState.result
     const { structStr, imageSrc, isLoading } = this.state
     const isCleanStruct = this.props.struct.isBlank()
-    return isLoading ? (
-      <div className={classes.loadingCirclesContainer}>
-        <LoadingCircles />
-      </div>
-    ) : this.isImageFormat(format) ? (
-      <div className={classes.imageContainer}>
-        {!isCleanStruct && (
-          <img
-            src={`data:image/${format}+xml;base64,${imageSrc}`}
-            alt={`${format} preview`}
+
+    if (isLoading) {
+      return (
+        <div className={classes.loadingCirclesContainer}>
+          <LoadingCircles />
+        </div>
+      )
+    } else if (this.isImageFormat(format)) {
+      return (
+        <div className={classes.imageContainer}>
+          {!isCleanStruct && (
+            <img
+              src={`data:image/${format}+xml;base64,${imageSrc}`}
+              alt={`${format} preview`}
+            />
+          )}
+        </div>
+      )
+    } else if (this.isBinaryCdxFormat(format)) {
+      return (
+        <div className={classes.previewBackground}>
+          <textarea
+            value="No preview available for this format"
```

Comment:
Hmmm, may be we don't need a readonly textarea. May be this is just worth showing some centered text in `<div>` a